### PR TITLE
Added trigger actions and executor

### PR DESCRIPTION
**Pull request by grafuls**

### CODE REVIEW

File: trigger/executor.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        if "ip_for_execute_command" is None then command will be executed on
+        the same host as the file
+        """
+        rc = None
+        if self.target:
+            self.initiate_executor(
+                self.target, remote_username, remote_password
+            )
+        logger.info(
+            "run command %s on ip %s", command_to_exec,
```
Comment (ILpinto) [inline trigger/executor.py]: run command %s on resource %s
We can also on on FQDN not just IPs

File: trigger/executor.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+from rrmngmnt.host import Host as HostResource
+from rrmngmnt.user import User
+
+DEFAULT_WAIT = 10
+DEFAULT_TIMEOUT = 240
+
+logger = logging.getLogger("executor")
+
+
+class Executor:
```
Comment (ILpinto) [inline trigger/executor.py]: It means Executor per resource ? if yes then resource and add actions to resource like: execute, get_status ...

Comment (ILpinto) [reply trigger/executor.py]: I mean that the Executor init resource and run the command on that resource.
So we can change it to Resource and add the execute action to it.
And maybe add more actions like get_status() ....

Comment (grafuls) [reply trigger/executor.py]: I see, I'll rename the executor to resource. Regarding the additional actions like get_status, I would like to implement asyncssh part of asyncio which will definitely come handy when expecting results without blocking execution, so leaving that for a different patch.

**Comment by grafuls** [comment]
Can we merge this already?

**Pull request merged**